src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -548,18 +548,13 @@ BoundExpression convertArrayElement(BoundExpression element, TypeSymbol elementT
             }
 
             var implicitReceiver = new BoundObjectOrCollectionValuePlaceholder(syntax, isNewInstance: true, targetType) { WasCompilerGenerated = true };
-            // PROTOTYPE: When generating a List<T>, should we use the well-known
-            // member List<T>.Add() rather than relying on lookup?
             var collectionInitializerAddMethodBinder = this.WithAdditionalFlags(BinderFlags.CollectionInitializerAddMethod);
             var builder = ArrayBuilder<BoundExpression>.GetInstance(node.Elements.Length);
             foreach (var element in node.Elements)
             {
                 var result = element switch
                 {
                     BoundBadExpression => element,
-                    // PROTOTYPE: Should spread elements support target type?
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=23&length=9)

Added open issue to test plan [https://github.com/dotnet/roslyn/issues/66418](https://github.com/dotnet/roslyn/issues/66418).